openslides/locale/fr/LC_MESSAGES/django.po

```diff
@@ -1894,245 +1925,262 @@ msgstr "Arrêtez de soumettre de nouvelles motions par les utilisateurs non-empl
 msgid "Allow to disable versioning"
 msgstr "Permettre de désactiver le versioning"
 
-#: motion/signals.py:76
+#: motion/signals.py:75
+msgid "Activate amendments"
+msgstr "Activer amendements"
+
+#: motion/signals.py:80
+msgctxt "Prefix for the identifier for amendments"
+msgid "A"
+msgstr "un"
```

Comment:
I think this translation is not correct. The Fench prefix should not be `un` but `A`, should it?